runbooks/VMCannotBeEvicted.md

```diff
@@ -0,0 +1,22 @@
+# VMCannotBeEvicted
+
+## Meaning
+
+This alert fires when the eviction strategy of a VM is set to "LiveMigration" but the VM is not migratable.
+
+## Impact
+
+Such a none-migratable VMs block node eviction which can impact operations such as node drain, updates, etc..
+
+## Diagnosis
+
+Check the eviction strategy and the Migratable status of the VMI
+
+- `kubectl get vmis  -o yaml `  
+   Search for evictionStrategy field. For example "evictionStrategy: LiveMigrate"
+
+- `kubectl get vmis  -o wide`  
+  Look at the "LIVE-MIGRATABLE" column of the output
+ 
+## Mitigation
+Change the VM to be migratable or set the evictionStrategy to shutdown
```

Comment:
Good point.  I added a description on how to inspect the VMI status & conditions to understand what was the exact reason the VM is not migratable. 
I preferred to give examples rather then putting the exact reasons that are used today to calculate the IsMigratable because this is a point in time status and some of the conditions might be changed over time.